src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -45,6 +45,10 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
     entering_time_min: [ null ],
     entering_time_max_enabled: [ false ],
     entering_time_max: [ null ],
+    entry_participant_type: [ false ],
+    entry_frozen_teams: [ false ],
+    entry_max_team_size: [ '', Validators.min(1) ],
+    entry_min_admitted_members_ratio: [ '' ],
```

Comment:
The form controls are named using snake case (probably to follow the backend model object), while the case convention for variables appears to be camel case. For further form developments, what casing should I use ?

Reply:
In particular, here, I see the the GET has been converted, but not the form nor the UPDATE.